tools/topology/sof-apl-pcm512x.m4

```diff
@@ -62,6 +73,20 @@ PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
 	1000, 0, 0,
 	48000, 48000, 48000)
 
+# DMIC01 passthrough capture pipeline 7 on PCM 4 using max 2 channels.
+# 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-passthrough-capture.m4,
```

Comment:
You could replace for DMIC pipelines pipe-passthrough-capture.m4 with pipe-volume-capture.m4. While pass-through has the merit of not altering DMIC PCM code values it's commented to be too silent. 

The unity gain is suitable to not distort by clipping rock concerts or street noise but captured office speech signal levels will be very low. With volume control and setting them to about +20 dB would give typical notebook capture level. Also capturing without gain with usual 16 bits precision will lose signal bits from LSB since the interface in HW is 24 bits.

Reply:
thanks, done